content/en/core/guides/android/devel-setup.md

```diff
@@ -0,0 +1,198 @@
+---
```

Comment:
I think this should be renamed to `development-setup.md` or similar, particularly because it actually shows up the URL and it'll help with findability.